srcpkgs/PrusaSlicer/patches/boost-1.81.patch

```diff
@@ -1,7 +1,7 @@
-Index: PrusaSlicer-2.4.2/src/slic3r/GUI/PrintHostDialogs.cpp
+Index: PrusaSlicer-2.5.0/src/slic3r/GUI/PrintHostDialogs.cpp
 ===================================================================
---- PrusaSlicer-2.4.2.orig/src/slic3r/GUI/PrintHostDialogs.cpp
-+++ PrusaSlicer-2.4.2/src/slic3r/GUI/PrintHostDialogs.cpp
+--- PrusaSlicer-2.5.0.orig/src/slic3r/GUI/PrintHostDialogs.cpp
++++ PrusaSlicer-2.5.0/src/slic3r/GUI/PrintHostDialogs.cpp
```

Comment:
This patch doesn't need to be refreshed.

Reply:
It does, the patch doesn't apply if i leave it like that. I might be wrong, let me try again.

Reply:
ok i will revert it, it doesn't seem to matter in this context.